UIKitDZ/UIKitDZ/AppetitViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  AppetitViewController.swift
+//  UIKitDZ
+//
+//  Created by Edvard on 30.06.2021.
+//
+
+import UIKit
+
+class AppetitViewController: UIViewController {
```

Comment:
```suggestion
final class AppetitViewController: UIViewController {
```
и для всех классов ниже если нет наследования